autoload/lsp/internal/semantic.vim

```diff
@@ -0,0 +1,216 @@
+let s:use_vim_textprops = lsp#utils#_has_textprops() && !has('nvim')
+let s:use_nvim_highlight = exists('*nvim_buf_add_highlight') && has('nvim')
+let s:textprop_cache = 'vim-lsp-semantic-cache'
+
+if s:use_nvim_highlight
+    let s:namespace_id = nvim_create_namespace('vim-lsp-semantic')
+endif
+
+if !hlexists('LspUnknownScope')
+    highlight LspUnknownScope gui=NONE cterm=NONE guifg=NONE ctermfg=NONE guibg=NONE ctermbg=NONE
```

Comment:
Let us make sure we document `LspUnknownScope`

Reply:
LspUnknownScope removed in [7a53e6e](https://github.com/prabirshrestha/vim-lsp/pull/1275/commits/7a53e6ecf12fac982406fb6c739a8e3a68941faa). Tokens are now assigned to custom highlight groups based on their token name.